docs/user/manifests/kubeapps-local-dev-values.yaml

```diff
@@ -30,3 +30,8 @@ ingress:
     nginx.ingress.kubernetes.io/proxy-read-timeout: "600"
     # Required for ingress-nginx 1.0.0 for a valid ingress.
     kubernetes.io/ingress.class: nginx
+packaging:
+  helm:
+    enabled: false
+  flux:
+    enabled: true
```

Comment:
Not sure we really want to use `flux` as the default plugin for dev?

Reply:
Woops - accidentally commited my local diff for testing here. Thanks.